question-servers/freeform.js

```diff
@@ -165,7 +165,12 @@ module.exports = {
             const resolvedElement = module.exports.resolveElement(elementName, context);
             const cwd = resolvedElement.directory;
             const controller = resolvedElement.controller;
-            const pythonArgs = [elementHtml, data];
+
+            let dataCopy = _.cloneDeep(data);
+            if ('base_url' in data.options) {
+                dataCopy.options.client_files_element_url = path.posix.join(data.options.base_url, 'elements', elementName, 'clientFilesElement');
```

Comment:
nit: a comment about why we use `path.posix` specifically would be good for future maintainers, as it might not be immediately obvious